modules/exploits/linux/local/autostart_persistence.rb

```diff
@@ -0,0 +1,60 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+    Rank = ExcellentRanking
```

Comment:
Typically Metasploit modules use two spaces for indentation.

Reply:
Ok, fixing